templates/createEventsPage.phtml

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
+    <title>Registered Applicants</title>
```

Comment:
This should be 'Academy Portal | Events'

Reply:
on it